docs/runbooks/CDIStorageProfilesIncomplete.md

```diff
@@ -1,27 +1,27 @@
+<!-- Edited by davozeni, 10 Nov 2022 -->
+
 # CDIStorageProfilesIncomplete
 
 ## Meaning
 
-An incomplete StorageProfile means that CDI will not be able to automatically infer PVC fields such as volumeMode, accessMode for the user's disk request.  
+This alert fires when any of the containerized data importer (CDI) storage profiles are incomplete.
 
-This alert fires when any of the StorageProfiles are incomplete, and thus not inferrable.
+When a storage profile is incomplete, CDI cannot automatically infer persistent volume claim (PVC) fields, such as `volumeMode` and  `accessModes`. These PVC fields are needed to successfuly complete a disk request from a user.
```

Comment:
Suggestion: "If a storage profile is incomplete, the CDI cannot infer persistent volume claim (PVC) fields, such as `volumeMode` and  `accessModes`, which are required to create a virtual machine (VM) disk."

Reply:
thank you for the suggestion. I applied it in the second commit.